readme.md

```diff
@@ -172,7 +172,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [nginx](https://github.com/fcambus/nginx-resources)
 - [Dropwizard](https://github.com/stve/awesome-dropwizard)
 - [Kubernetes](https://github.com/ramitsurana/awesome-kubernetes)
-
+- [Vorpal](https://github.com/vorpaljs/awesome-vorpaljs)
```

Comment:
It's not really "back-end development".

It's also inconsistent that the name of the list is `Vorpal`, while the URL is `vorpaljs`.